src/components/compound/DlSelect/components/DlSelectOption.vue

```diff
@@ -4,7 +4,17 @@
         :class="identifierClass"
         style="width: 100%"
     >
+        <div
+            v-if="readonly"
+            :class="[{ 'readonly-option': true }, { capitalized }]"
+            :style="`padding-left: ${10 + depth * 30}px;`"
+        >
+            <slot>
```

Comment:
why div and not dl-list-item ?

Reply:
Because I don't want/need all of the dl-list-item functionalities (hover state, text formatting, etc..).
It's just a simple string presenter.